ccmlib/node.py

```diff
@@ -680,16 +680,20 @@ def stop(self, wait=True, wait_other_notice=False, gently=True):
         else:
             return False
 
-    def wait_for_compactions(self):
+    def wait_for_compactions(self, timeout=600):
```

Comment:
Given we aren't tracking seconds, but rather loop iterations, could you make this default smaller?

Or could you actually track time passed?